openshift/release/update-to-head.sh

```diff
@@ -10,7 +10,7 @@ TODAY=`date "+%Y%m%d"`
 
 # Reset release-next to upstream/master.
 git fetch upstream master
-git checkout upstream/master -B release-next
+git checkout upstream/master --no-track -B release-next
 
```

Comment:
ah I had  something like this in my local update-to-head.sh i didn't push it since i tought it was because of some mess in my git repo: 

```
git checkout -B release-next
git reset--hard upstream/master
```